finagle-stats/src/test/scala/com/twitter/finagle/stats/MetricsBucketedHistogramTest.scala

```diff
@@ -132,15 +132,63 @@ class MetricsBucketedHistogramTest extends FunSuite {
 
       // call .snapshot() to recompute counts
       h.snapshot()
-      val countsSnap0 = details.counts
-      assert(countsSnap0 == Seq(BucketAndCount(1, 2, 1)))
+      assert(details.counts == Seq(BucketAndCount(1, 2, 1)))
 
       h.add(Int.MaxValue)
       // roll to window 2 (this should make data A visible after a call to snapshot)
       roll()
-      val countsSnap1 = details.counts
-      assert(countsSnap1 == Seq(BucketAndCount(1, 2, 1),
+      assert(details.counts == Seq(BucketAndCount(1, 2, 1),
         BucketAndCount(2137204091, Int.MaxValue, 1)))
+
+      // Advance several minutes without snapshotting (i.e. to
```

Comment:
The test name, _histogram snapshot respects refresh window_, seemed appropriate...  i'll split it out if that's clearer, though.